src/v2/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryFigure.tsx

```diff
@@ -31,27 +35,42 @@ const ArticleZoomGalleryFigure: FC<ArticleZoomGalleryFigureProps> = ({
 
   const xl = {
     img: resized(src, { width: 2000, height: 2000 }),
-    media: "(min-width: 1720px)",
+    media: {
+      query: "(min-width: 1720px)",
+      preload: "(min-width: 1720px)",
+    },
   }
 
   const lg = {
     img: resized(src, { width: 1440, height: 1440 }),
-    media: "(min-width: 1232px)",
+    media: {
+      query: "(min-width: 1232px)",
+      preload: "(min-width: 1232px) and (max-width: 1719px)",
+    },
   }
 
   const md = {
     img: resized(src, { width: 1024, height: 1024 }),
-    media: "(min-width: 896px)",
+    media: {
+      query: "(min-width: 896px)",
+      preload: "(min-width: 896px) and (max-width: 1231px)",
+    },
   }
 
   const sm = {
     img: resized(src, { width: 767, height: 767 }),
-    media: "(min-width: 767px)",
+    media: {
+      query: "(min-width: 767px)",
+      preload: "(min-width: 767px) and (max-width: 895px)",
+    },
   }
 
   const xs = {
     img: resized(src, { width: 450, height: 450 }),
-    media: "(max-width: 766px)",
+    media: {
+      query: "(max-width: 766px)",
+      preload: "(max-width: 766px)",
```

Comment:
😓 😓 

It'd be nice to key this off of `THEME.breakpoints` if possible, but all good if there's not a reason to abstract right now.